test/integration/guest_user_test.rb

```diff
@@ -67,4 +67,24 @@ def setup
     visit items_path
     refute page.has_content?("Create Category")
   end
+
+  test "an unauthorised user can view a single item's page" do
+    item = Item.create(title: 'coffee', description: 'black nectar of the gods', price: 1200)
```

Comment:
Useless assignment to variable - `item`.<br>Line is too long. [93/80]<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.